Functions.Templates/Templates-v2/EventHubTrigger-Python/template.json

```diff
@@ -0,0 +1,204 @@
+{
+    "author": "Gavin Aguiar",
+    "name": "EventHub trigger",
+    "description": "$EventHubTrigger_description",
+    "programmingModel": "v2",
+    "language": "python",
+    "jobs": [
+        {
+            "name": "Create New Project with EventHubTrigger function",
+            "input" : {
+                "userCommand" : "createNewApp",
+                "assignTo" : "$(APP_FILENAME)"
+            },
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionApp",
+                "replaceTokensInText_FunctionApp",
+                "writeFile_FunctionApp",
+                "showMarkdownPreview"
+            ]
+        },
+        {
+            "name": "Add EventHubTrigger function to the main file",
+            "actions": [
+                "getFunctionName",
+                "getEventHubName",
+                "getConnnectionString",
+                "readFileContent_FunctionBody",
+                "replaceTokensInText_FunctionBody",
+                "appendFileContent_FunctionApp"
+            ]
+        },
+        {
+            "name": "Create New Blueprint file",
+            "input" : {
+                "userCommand" : "createBluePrint",
+                "assignTo" : "$(BLUEPRINT_FILENAME)"
+            },
+            "actions": [
+                "getBluePrintFileName",

```

Comment:
Action `getBlueBrintFileName` is not defined in template.json - did you mean `getBluePrintName`?

Reply:
Added getBluePrintFileName